tests/apps/search/test_index_manager.py

```diff
@@ -155,165 +158,129 @@ def get_es_documents(self, index, action="index"):
-                    },
-                    {
-                        "add": {
-                            "index": "richie_organizations_created_index",
-                            "alias": "richie_organizations",
-                        }
-                    },
-                    {
-                        "add": {
-                            "index": "richie_persons_created_index",
-                            "alias": "richie_persons",
-                        }
-                    },
-                    {
-                        "remove": {
-                            "index": "richie_categories_forgotten",
-                            "alias": "richie_categories",
-                        }
-                    },
-                    {
-                        "remove": {
-                            "index": "richie_categories_previous",
-                            "alias": "richie_categories",
-                        }
-                    },
-                    {
-                        "remove": {
-                            "index": "richie_courses_forgotten",
-                            "alias": "richie_courses",
-                        }
-                    },
-                    {
-                        "remove": {
-                            "index": "richie_courses_previous",
-                            "alias": "richie_courses",
-                        }
-                    },
-                    {
-                        "remove": {
-                            "index": "richie_organizations_forgotten",
-                            "alias": "richie_organizations",
-                        }
-                    },
-                    {
-                        "remove": {
-                            "index": "richie_organizations_previous",
-                            "alias": "richie_organizations",
-                        }
-                    },
-                    {
-                        "remove": {
-                            "index": "richie_persons_forgotten",
-                            "alias": "richie_persons",
-                        }
-                    },
-                    {
-                        "remove": {
-                            "index": "richie_persons_previous",
-                            "alias": "richie_persons",
-                        }
-                    },
-                ]
-            }
-        )
-        self.indices_client.delete.assert_called_with(
-            ignore=[400, 404], index="richie_orphan"
-        )
+        # Create all our indices from scratch
+        # Use a mocked timezone.now to  check the names of our indices as they include a datetime
+        creation1_datetime = datetime(2010, 1, 1, tzinfo=timezone.utc)
+        creation1_string = creation1_datetime.strftime("%Y-%m-%d-%Hh%Mm%S.%fs")
+        with mock.patch.object(timezone, "now", return_value=creation1_datetime):
+            regenerate_indexes(None)
```

Comment:
You should probably search and replace `indexes` for `indices` on the whole code base! ^^